stories/autocomplete.stories.ts

```diff
@@ -30,8 +32,10 @@ storiesOf('Basics|Autocomplete', module).add(
           widgetParams.container.appendChild(list);
         }
 
-        widgetParams.container.querySelector('input').value = currentRefinement;
-        widgetParams.container.querySelector('ul').innerHTML = indices
+        widgetParams.container.querySelector(
```

Comment:
You can type the selectors because `querySelector` is generic. Then you might not need the `!`.

```suggestion
        widgetParams.container.querySelector<HTMLInputElement>(
```

Reply:
I don't think so, the type returned in querySelector can only be truly known at run time, therefore it can be *indeed* null.

Reply:
Right—you can still type it as `HTMLInputElement` to have type safety for `value` though.